images/federation/Dockerfile

```diff
@@ -0,0 +1,12 @@
+#
+# This is the OpenShift Origin Federation image, used for running the
+# federation apiserver and controller manager components.
+#
+# The standard name for this image is openshift/origin-federation
+#
+FROM openshift/origin-base
```

Comment:
>  What's the simplest acceptable solution to getting this image built for 3.6?

To be clear... my comment is prefixed with that "IF" for a reason. If we are shipping this, we need to RPM install it. If we are not, just keep doing what you're doing.